AzureMonitorAgent/ama_tst/modules/install/check_ama.py

```diff
@@ -0,0 +1,92 @@
+import re
+import requests
+from bs4 import BeautifulSoup
+
+from error_codes import *
+from errors      import error_info, get_input
+from helpers     import get_package_version
+
+AMA_URL = 'https://docs.microsoft.com/en-us/azure/azure-monitor/agents/azure-monitor-agent-extension-versions'
+
+def get_curr_ama_version(ama_version):
+    try:
+        r = requests.get(AMA_URL)
+        html_doc = BeautifulSoup(r.text, 'html.parser')
+        rows = html_doc.find("table").find("tbody").find_all("tr")
+
+        for row in rows:
+            cells = row.find_all("td")
+            version = cells[3].get_text()
+            version = re.sub('[A-Za-z ]+', '', version)
+            if (version == ''):
+                continue
+            if (comp_versions_ge(version, ama_version)):
+                return (version, None)
+
+    except Exception as e:
+        return (None, e)
+    return (None, None)
+
+# compare two versions, see if the first is newer than / the same as the second
+def comp_versions_ge(version1, version2):
+        versions1 = [int(v) for v in version1.split(".")]
+        versions2 = [int(v) for v in version2.split(".")]
+        for i in range(max(len(versions1), len(versions2))):
+            v1 = versions1[i] if i < len(versions1) else 0
+            v2 = versions2[i] if i < len(versions2) else 0
+            if v1 > v2:
+                return 1
+            elif v1 < v2:
+                return -1
+        return 0
+
+def ask_update_old_version(ama_version, curr_ama_version):
+    print("--------------------------------------------------------------------------------")
+    print("You are currently running AMA Verion {0}. There is a newer version\n"\
+          "available which may fix your issue (version {1}).".format(ama_version, curr_ama_version))
+    answer = get_input("Do you want to update? (y/n)", (lambda x : x.lower() in ['y','yes','n','no']),\
+                       "Please type either 'y'/'yes' or 'n'/'no' to proceed.")
+    # user does want to update
+    if (answer.lower() in ['y', 'yes']):
+        print("--------------------------------------------------------------------------------")
+        print("Please head to the Github link below\n")
```

Comment:
I'd just remove the Github link part - all of the instructions on how to update can be found in the MSFT docs page